modAionImpl/src/org/aion/zero/impl/pendingState/AionPendingStateImpl.java

```diff
@@ -280,140 +287,102 @@ private void addTransactionsFromCacheOrBackup(List<AionTransaction> transactions
         List<TxResponse> txResponses = new ArrayList<>();
 
         for (AionTransaction tx : transactions) {
-            BigInteger txNonce = tx.getNonceBI();
-            BigInteger bestPSNonce = bestPendingStateNonce(tx.getSenderAddress());
-            AionAddress txFrom = tx.getSenderAddress();
-
-            int cmp = txNonce.compareTo(bestPSNonce);
+            BigInteger bestPendingStateNonce = bestPendingStateNonce(tx.getSenderAddress());
+            int cmp = tx.getNonceBI().compareTo(bestPendingStateNonce);
```

Comment:
The code would be easier to read if after computing this variable we set flags with meaningful names and use those in the `if` statements below.